src/containers/views/DataFileEdit.js

```diff
@@ -95,11 +118,16 @@ export class DataFileEdit extends Component {
         className="single">
         {errors.length > 0 && <Errors errors={errors} />}
         <div className="content-header">
-          <Breadcrumbs splat={filename} type="datafiles" />
+          <Breadcrumbs splat={directory || ""} type="data files" />
         </div>
 
         <div className="content-wrapper">
           <div className="content-body">
+            <InputPath
+              onChange={onDataFileChanged}
+              type="data files"
+              path={path} // TODO: Support using `relative_path` from API instead
```

Comment:
With the current behavior, you can convert the data file into a static file by dropping it outside the `data_dir` :grinning: 
```
/_data/data_file.yml => /foo/data_file.yml
```